workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -692,6 +692,19 @@ pre[class*='language-'] {
 	}
 }
 
+.search-banner {
+	&::after {
+		top: 155px !important;
+	}
+
+	&::before {
+		top: 198px !important;
+	}
+
+	height: 198px !important;
+	padding-top: 24px !important;
```

Comment:
I have been trying to avoid padding in the css if we can do it in the fragments itself. Also if we do need padding, we should be using the --spacer variables to keep them consistent